test/sanity/validate-modules/module_args.py

```diff
@@ -75,7 +77,26 @@ def setup_env(filename):
                 del sys.modules[k]
 
 
-def get_argument_spec(filename):
+def get_ps_argument_spec(filename):
+    # This uses a very small skeleton of Ansible.Basic.AnsibleModule to return the argspec defined by the module. This
+    # is pretty rudimentary and will probably require something better going forward.
+    cmd = ['test/sanity/validate-modules/ps_argspec.ps1', filename]
```

Comment:
I'd calculate this path relative to `__file__` instead of assuming the cwd (in case someone runs it without using ansible-test for example).